### PR TITLE
ci-operator: add etcd-operator

### PR DESCRIPTION
**Pull request by hexfusion**
The test suite is not compatible with ci-operator without a fair amount of work. We will need to manually do functional testing.

### CODE REVIEW

**Comment by hexfusion** [comment]
@stevekuznetsov PTAL

File: ci-operator/config/openshift/etcd-operator/openshift-etcd-operator-openshift-4.1.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tag_specification:
+  name: "4.1"
+  namespace: ocp
+tests:
+- as: unit
+  commands: |
+    curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
```
Comment (stevekuznetsov) [inline ci-operator/config/openshift/etcd-operator/openshift-etcd-operator-openshift-4.1.yaml]: Let's add `dep` to the installs in the base image(s) for recent Go versions so you don't need to do this every test

Comment (stevekuznetsov) [reply ci-operator/config/openshift/etcd-operator/openshift-etcd-operator-openshift-4.1.yaml]: https://github.com/openshift/release/blob/master/projects/origin-release/golang-1.10/Dockerfile#L32

**Comment by ehashman** [comment]
Can this be closed? Not sure why it's got a sig/azure label.

**Comment by hexfusion** [comment]
/close 

this needs to be rethought, we are probably going to move to use the ansible ha etcd operator here.

**Pull request closed without merging**